templates/core/cafe.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Sharespots</title>
+</head>
+<body>
+    {% if cafe %}
+        <h1>{{ cafe.name }}</h1>
+        <p>{{ cafe.address_1 }}</p>
+        <p>{{ cafe.address_2 }}</p>
+        <p>{{ cafe.post_code }}</p>
+        <p>{{ cafe.description }}</p>
+    {% else %}
+        <h1>No cafe for this id.</h1>
```

Comment:
this can be removed once you are using djangos get_object_or_404

Reply:
Okey doke.